app/services/pipeline_service/events/http_client.rb

```diff
@@ -2,12 +2,12 @@ module PipelineService
   module Events
```

Comment:
This may by why Graf wasn't receiving a message before.  I'm using explicit parameters in this wrapper class instead of splat crap